gui/wxpython/datacatalog/toolbars.py

```diff
@@ -69,7 +72,9 @@ def _toolbarData(self):
                                      ("reloadMapset", icons["reloadMapset"],
                                       self.parent.OnReloadCurrentMapset),
                                      ("lock", icons['locked'],
-                                      self.OnSetRestriction, wx.ITEM_CHECK)
+                                      self.OnSetRestriction, wx.ITEM_CHECK),
+                                     ("createGrassDb", icons['locked'],
+                                      self.parent.OnCreateGrassDb)
```

Comment:
This should call something like OnAddGrassDB which opens a standard file dialog to select a directory and then inserts the new node and reloads data.